app/views/layouts/_footer.html.erb

```diff
@@ -1,13 +1,13 @@
 <footer class="footer">
   <small>
-    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+    The <a href="http://www.railstutorial.org/"><%= t ".title_page" %></a>
```

Comment:
I18n, use link_to instead of a tag, move "http://www.railstutorial.org/ to settings